css/style.css

```diff
@@ -60,7 +61,8 @@
 
 .entry-content h3.heading a,
 .entry-content h3.heading i {
-	color: #338bc5 !important;
```

Comment:
Sadly, I think this important needs to stay so the section headers at the bottom of the homepage don't turn purple as visited links. Also, I think the font-size can be removed.

Reply:
ok code climate may flag the 'important'